go/libkb/proof_support_coinbase.go

```diff
@@ -92,21 +85,19 @@ func coinbaseSettingsURL(s string) string {
 	return coinbaseUserURL(s) + "#settings"
 }
 
-func (t CoinbaseServiceType) NormalizeUsername(s string) (ret string, err error) {
-	ret = strings.ToLower(s)
+func (t CoinbaseServiceType) NormalizeUsername(s string) (string, error) {
+	if !regexp.MustCompile(`^(?i:[a-z0-9_]{2,16})$`).MatchString(s) {
```

Comment:
Also, compile it once and store the compiled regex instead of compiling it every time this is called.